pkg/task/common.go

```diff
@@ -405,7 +407,27 @@ func ReadBackupMeta(
 	}
 	metaData, err := s.ReadFile(ctx, fileName)
 	if err != nil {
-		return nil, nil, nil, errors.Annotate(err, "load backupmeta failed")
+		if gcsObjectNotFound(err) {
+			// change gcs://bucket/abc/def to gcs://bucket/abc and read defbackupmeta
+			parsedURL, err := storage.ParseRawURL(cfg.Storage)
+			if err != nil {
+				return nil, nil, nil, errors.Trace(err)
+			}
+			newPrefix, file := filepath.Split(parsedURL.Path)
```

Comment:
I mean we should just use `path.Split` here since the path comes from a URL which always use `/` as separator, instead of `filepath.Split` which the separator depends on the OS